test/test16.c

```diff
@@ -72,17 +55,42 @@ struct my_struct Calls("free", "x") does_something(struct my_struct S Calls("fre
 }
 */
 
+MustCall(STRUCT("my_struct") AT_FIELD("x"), METHODS("free"))
+MustCall(STRUCT("my_struct") AT_FIELD("y"), METHODS("free"))
+MustCall(STRUCT("my_struct") AT_FIELD("x"), METHODS("free"))
+MustCall(STRUCT("my_struct") AT_FIELD("y"), METHODS("free"))
+Calls(FUNCTION("does_free") AT_PARAMETER("1"), METHODS("free"))
+Calls(FUNCTION("creates_obligation") AT_PARAMETER("1"), METHODS("free"))
+Calls(FUNCTION("creates_obligation") AT_PARAMETER("2") AT_FIELD("x"), METHODS("free"))
+MustCall(FUNCTION("creates_obligation") AT_RETURN, METHODS("free")) 
+Calls(FUNCTION("does_something") AT_RETURN AT_FIELD("x"), METHODS("free"))
+MustCall(STRUCT("my_struct") AT_FIELD("x"), METHODS("free"))
+MustCall(STRUCT("my_struct") AT_FIELD("y"), METHODS("free"))
```

Comment:
These two are also duplicates with lines 58-59.

Reply:
Yeah those are typos; I meant to refer to the other 2 structs